packages/sindarian-ui/src/globals.css

```
@@ -325,41 +327,92 @@
 
-/* Scrollbar — thin translucent thumb for scrollable layout containers */
-@layer base {
-  * {
-    scrollbar-width: thin;
-    scrollbar-color: var(--color-shadcn-300) transparent;
-  }
+[data-sonner-toast] {
+  --normal-bg: hsl(var(--card));
+  --normal-text: hsl(var(--foreground));
+  --normal-border: hsl(var(--border));
+  padding: 24px 32px 24px 24px !important;
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)